GUI.cpp

```diff
@@ -1343,7 +1343,6 @@ int loadImgFile(const char* filename)
         }
         int op, src, dst, alpha_op, alpha_src, alpha_dst;
         al_get_separate_blender(&op, &src, &dst, &alpha_op, &alpha_src, &alpha_dst);
-        ALLEGRO_BITMAP* currentTarget = al_get_target_bitmap();
```

Comment:
Confirmed that this has no side effects: https://github.com/liballeg/allegro5/blob/acf307bcf75192da30ce769c8ca65f152667311c/src/tls.c#L509-L516